### ISSUE TITLE
Add DFU flashing to web config

### ISSUE THREAD

**Issue by ASleepyCat** [HWXLR8/beef-board#108]
AFAIK no one else has done this in JS? Can't find any existing (open source) examples on Google.

**Comment by ASleepyCat** [comment]
`beef-tool` can be kept as an alternative method in case someone doesn't want to or can't use a browser with WebHID support.

**Comment by HWXLR8** [comment]
Yeah, as someone who exclusively uses firefox (no hid support), I'd rather use beef tool than install another web browser
